gdal/frmts/wms/minidriver_tiled_wms.cpp

```diff
@@ -237,26 +265,22 @@ static void FindChangePattern(char *cdata, char **substs, char **keys, CPLString
                 // But is the match for the key position?
                 char *found_key = nullptr;
                 const char *found_value = CPLParseNameValue(substs[sub_number], &found_key);
-                if (found_key != nullptr && EQUAL(found_key, key))
-                {  // Should exits in the request
+                if (found_key != nullptr && EQUAL(found_key, key)) {
+                    // Should exist in the request
```

Comment:
It is
- mixture of code style change with functional change (in other lines)
- code style change to a style different than the majority of GDAL codebase

and both are discouraged and undesired.

Reply:
OK